README.md

```diff
@@ -1,20 +1,21 @@
-# gateway-sidecar
+# Gateway Sidecar
 
 The Slice VPN Gateway is a slice network service component that provides a secure VPN tunnel between any two clusters that are a part of the slice. 
```

Comment:
As per Eric's suggestion, add more info on the purpose. 

Something like - Installing gateway-sidecar provides a Slice VPN Gateway, a slice network... 

or check with the dev team for the purpose. 

cc @rajendra-avesha / @gourishkb

Reply:
@veenamj-avesha  This PR was raised to fix the hyperlinks. I will work on the introduction in a separate PR. As dev team is busy with the release now